docs/content/en/docs/getting-started/observability/_index.md

```diff
@@ -1,6 +1,244 @@
+file defines the `Namespace` resource for our example.
+
+### Verify that your cluster is ready
+
+Run the following command to ensure that your Kuberetes cluster
+is ready to complete this exercise by running:
+
+```shell
+kubectl get pods -n keptn-lifecycle-toolkit-system
+```
+
+You should see pods for the following components:
+- certificate-operator (or another cert manager)
+- lifecycle-operator
+- scheduler
+- otel-collector
+- metrics-operator
+
+### Annotate Deployment resource
+
+Follow the instructions in
+[Annotate workload](../../implementing/integrate/#basic-annotations)
+to apply basic annotations to your `Deployment` resource.
+
+The
+[simplenode-dev-deployment.yaml](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/simplenode-dev-deployment.yaml/)
+file defines the `Deployment` resource for our example.
+
+The example file also includes annotations for
+pre- and post-deployment activities.
+We will discuss those in a separate exercise.
+
+## Define the Keptn application
+
+A Keptn application defines the workloads
+to be included in your Keptn Application.
+We will use the application discovery feature
+to automatically generate a Keptn Application
+that includes all workloads on the cluster,
+regardless of the tools being used.
+
+A Keptn application aggregates multiple workloads
+that belong to a logical app into a single
+You can create the YAML file to define the
+[KeptnApp](../../yaml-crd-ref/app.md)
+resource manually
+but the easier approach is to let KLT create this definition for you.
+This requires that you annotate all your workloads
+(`Deployments`, `Stateful Sets`, `DaemonSets`, and `ReplicaSets`
+as described in
+[Use Keptn automatic app discovery](../../implementing/integrate.md/#use-keptn-automatic-app-discovery).
+
+## DORA metrics
+
+DORA metrics are an industry-standard set of measurements;
+see the following for a description:
+
+* [What are DORA Metrics and Why Do They Matter?](https://codeclimate.com/blog/dora-metrics)
+* [Are you an Elite DevOps Performer?  Find out with the Four Keys Project](https://cloud.google.com/blog/products/devops-sre/using-the-four-keys-to-measure-your-devops-performance)
+
+DORA metrics provide information such as:
+
+* How many deployments happened in the last six hours?
+* Time between deployments
+* Deployment time between versions
+* Average time between versions.
+
+The Keptn Lifecycle Toolkit starts collecting these metrics
+as soon as you annotate the `Deployment` resource.
+
+TODO: Are Dora metrics collected when KLT is enabled or only
+      when the `Deployment` resource is annotated?
+
+TODO: How to observe DORA metrics without a dashboard.
+      The info about observing Keptn metrics seems specific to those;
+      if it's the same commands, what are the names of each DORA
+      metric to specify?
+
+## Using OpenTelemetry
+
+The Keptn Lifecycle Toolkit extends the Kubernetes
+pod scheduler to create OpenTelemetry data
```

Comment:
To be more future proof since soon-ish we won't use anymore the scheduler.
```suggestion
The Keptn Lifecycle Toolkit extends the Kubernetes
primitives to create OpenTelemetry data
```